content/en/synthetics/guide/api_test_timing_variations.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Understanding API test timings and troubleshooting their variations
+kind: documentation
+description: Understanding API test timings and troubleshooting their variations
+aliases:
+- /synthetics/api_test_timing_variations
+further_reading:
+- link: "https://docs.datadoghq.com/synthetics/metrics/#api-tests"
+  tag: "Documentation"
+  text: "Synthetics API test timing metrics"
+---
+
+
+## Overview
+
+Synthetics API tests collect several [**timing metrics**][1] which can be used to identify any bottlenecks in the communication between the client and your server. 
+
+## Collected Timings
+
+
+The metrics collected by Synthetics measure the following timings : 
+
+
+**1. Redirection time**
+
+The `synthetics.http.redirect.time` metric measures the total time spent in redirects. All other network timings (DNS resolution, TCP connection, etc.) correspond to the last request. For example, if an HTTP test (that has the **Follow Redirects** option turned on) first loads page A in 35 ms in total, which redirects to page B that loads in 40ms in total, which redirects to page C, the redirect timing equals 35 + 40 = 75 ms. The load of page C is then split among the various timings (DNS resolution, TCP connection, etc.).
```

Comment:
Maybe add link to /synthetics/api_tests/http_tests?tab=requestoptions#define-request for follow redirects